bin/cli.js

```diff
@@ -41,14 +43,18 @@ program
       const mineAddresses = await web3.eth.getAccounts()
       mineAddress = mineAddresses.length && mineAddresses[0]
     }
-
-    const ipfsUrl = options.ipfsUrl || 'http://localhost:5001'
-    // conver to object format as expected by ipfs-api
-    const ipfsUrlOpts = url.parse(ipfsUrl)
-    ipfsUrlOpts.host = ipfsUrlOpts.hostname // use only hostname w/o port
-    const ipfs = ipfsAPI(ipfsUrlOpts)
-
-    app.checkForModels(mineAddress, contractAddress, web3, ipfs)
+    if (isIPFS.ipfsUrl(options.ipfsUrl)) {
+      const ipfsUrl = options.ipfsUrl || 'http://localhost:5001'
```

Comment:
Fallback to default should happen outside the if statement. Otherwise it still never activate